### PR TITLE
fix: Enable additional GKE outputs

### PR DESCRIPTION
**Pull request by gls4**
This pr outputs additional information about the GKE cluster.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -28,37 +10,70 @@ output "bucket_queue_name" {
...
+output "service_account" {
+  value       = module.service_accounts.service_account
+  description = "Weights & Biases service account used to manage resources."
+}
+output "url" {
+  value       = local.url
+  description = "The URL to the W&B application"
+}
+
+
```
Comment (kimjyhello) [inline outputs.tf:72]: very nit: are these extra spaces put in there intentionally?

**Review by kimjyhello — APPROVED** [review]

File: outputs.tf (hunk 1 above)
Comment (gls4) [reply outputs.tf:72]: No. Heh.

**Comment by jsbroks** [comment]
This PR is included in version 1.20.1 :tada:

File: outputs.tf [hunk 2]
```diff
@@ -28,37 +10,70 @@ output "bucket_queue_name" {
...
+
+
```
Comment (elainaRenee) [inline outputs.tf:74]: @gls4 can we remove the extra spaces? 🙏

File: modules/app_gke/outputs.tf [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
+  sensitive = true
+  value = google_container_cluster.default.master_auth.0.client_key
+}
+output "cluster_endpoint" {
+  value = google_container_cluster.default.endpoint
+}
+output "cluster_id" {
+  value = google_container_cluster.default.id
+}
 output "cluster_name" {
```
Comment (elainaRenee) [inline modules/app_gke/outputs.tf:22]: nit: spacing is inconsistent in this file too

**Pull request merged**